src/main/java/ch/njol/skript/expressions/ExprExperienceChangeReason.java

```diff
@@ -0,0 +1,40 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.EventValueExpression;
+import org.bukkit.event.Event;
+import org.bukkit.event.player.PlayerExpCooldownChangeEvent.ChangeReason;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Experience Change Reason")
+@Description({
+	"The <a href='classes.html#experiencechangereason'>experience change reason</a> within in an" +
+	"<a href='events.html#experience cooldown change event'>experience cooldown change event</a>."
+})
+@Examples({
+	"on player experience cooldown change:",
+		"\tif xp change reason is plugin:",
+			"\t\t#Changed by a plugin",
+		"\telse if xp change reason is orb pickup:",
+			"\t\t#Changed by picking up xp orb"
+})
+@Since("INSERT VERSION")
+public class ExprExperienceChangeReason extends EventValueExpression<ChangeReason> {
```

Comment:
I feel like this should all be more obviously referencing the fact that its in regard to the cooldown change, not the experience change.